website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md

```diff
@@ -68,5 +68,8 @@ If your account is not connected, you’ll see "This account is not linked to a
 
 ### Errors importing a repository
 If you do not see your repository listed, double-check that:
-- You are a maintainer on that repository (only users with a maintainer or owner permissions can set up repository connections
 - Your repository is in a Gitlab group you have access to
+
+If you do see your repository listed but after selecting it, it's stuck loading/errors, double-check that:
+- You are a maintainer on that repository (only users with a maintainer or owner permissions can set up repository connections
```

Comment:
missing a ")" at the end of this line. 

Other than that, looks great to me!

Reply:
That's so weird...I see it?

Reply:
I just fixed it so it's possible you were on an older page?